### ISSUE TITLE
Wrong version in conda-forge package

### ISSUE THREAD

**Issue by tvery42** [bio-phys/MDBenchmark#171]
I installed mdbenchmark with the following command:
```
   conda create -n mdbenchmark -c conda-forge mdbenchmark=3.0.1 python=3.7.6
```

But the version I got seems to be a 2.0.1:

```
$ mdbenchmark --version
mdbenchmark, version 2.0.1
```

The conda version that i used  4.8.5.

```
$ conda list
 # Name  
mdbenchmark               3.0.1            py37hc8dfbb8_0    conda-forge
```

**Comment by mimischi** [comment]
Hey @tvery42, thanks for the notification! Indeed something changed with conda-forge and the package is now only available on the "cf-staging" channel. Could you test the following:

```
   conda create -n mdbenchmark -c cf-staging mdbenchmark=3.0.1 python=3.7.6
```

I need to figure out how to push the package from `cf-staging` to `conda-forge` or change the documentation accordingly.

**Comment by tvery42** [comment]
It seems the the package is not available.

```
$ conda create -n mdbenchmark -c cf-staging mdbenchmark=3.0.1 python=3.7.6
Collecting package metadata (current_repodata.json): done
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - mdbenchmark=3.0.1

Current channels:

  - https://conda.anaconda.org/cf-staging/linux-64
  - https://conda.anaconda.org/cf-staging/noarch
  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch
```

**Comment by mimischi** [comment]
@tvery42 I talked with the conda-forge team and they told me that the `cf-staging` should not be used, so disregard my message above.

I have just tested again on two machines (Linux + macOS) and was able to install version 3.0.1 without any issues:

```bash
conda create -n mdb -c conda-forge python==3.7.6 mdbenchmark
[...]
mdbenchmark --version
mdbenchmark, version 3.0.1
```

Could you please try again? Not sure if something changed, but it seems to work for me. As a troubleshooting procedure, could you run `which mdbenchmark` when you have _not_ activated any conda environments? Maybe you installed the package in your global environment through pip? I had exactly this case while testing right now.

**Comment by tvery42** [comment]
It works fine now!

When I had the problem it was the first time I installed mdbenchmark.
I needed some features only available in version 3.0.1, so after posting this report I installed it via pip. But this was version 3.0.1 for sure.

**Comment by mimischi** [comment]
Alright, thank you for getting back! I'll be closing the issue now. Feel free to open another one if you experience any other issues!
